cmd/evmosd/root.go

```diff
@@ -303,7 +303,7 @@ func (a appCreator) appExport(
 func initTendermintConfig() *tmcfg.Config {
 	cfg := tmcfg.DefaultConfig()
 	cfg.Consensus.TimeoutCommit = time.Second
-	cfg.Mempool.Version = tmcfg.MempoolV1
+	cfg.Mempool.Version = tmcfg.MempoolV0
```

Comment:
Btw will this force nodes, rather than validators, to use v1? Normal nodes no need this one

Reply:
I meant before revert code

Reply:
Btw2, just remove the line is done, isn't it? Who want it, enable in the config

Reply:
@VictorTrustyDev we did some benchmarks and the v1 version severely affects performance of the nodes. It's better to state 
```suggestion
	// use v0 since v1 severely impacts the node's performance
	cfg.Mempool.Version = tmcfg.MempoolV0
```

Reply:
I see, this comment points out the idea why hard-coded